CalculatorExercises/1-CalculatorBasic/Main.java

```diff
@@ -0,0 +1,27 @@
+import java.util.*;
```

Comment:
avoid importing everything with *, check what classes you need and import them separately